CONTRIBUTING.md

```diff
@@ -34,19 +34,36 @@ git checkout -b <branch name>
 
 
 ## Submitting a Pull Request 
 
-1. Please add yourself to the Mattermost [approved contributor list](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true) prior to submitting by completing the [contributor license agreement](http://www.mattermost.org/mattermost-contributor-agreement/). 
+1. Please add yourself to the Mattermost [approved contributor list][approved-contributor-list] prior to submitting by completing the [contributor license agreement][cla].
 
-2. When you submit your pull request please make it against `master` and include the Ticket ID at the beginning of your pull request comment, followed by a colon
+2. All contributors are required to register on the [Mattermost core team channel][core-channel-invite].
 
-  - For example, for a ticket ID `PLT-394` start your comment with:  `PLT-394:`. See [previously closed pull requests](https://github.com/mattermost/platform/pulls?q=is%3Apr+is%3Aclosed) for examples
+3. When you submit your pull request please make it against `master` and include the Ticket ID at the beginning of your pull request comment, followed by a colon
 
-3. Once submitted, your pull request will be checked via an automated build process and will be reviewed by at least two members of the Mattermost core team, who may either accept the PR or follow-up with feedback. It would then get merged into `master` for the next release
+  - For example, for a ticket ID `PLT-394` start your comment with:  `PLT-394:`. See [previously closed pull requests][ticket-examples] for examples
+
+4. Once submitted, your pull request will be checked via an automated build process and will be reviewed by at least two members of the Mattermost core team, who may either accept the PR or follow-up with feedback. It would then get merged into `master` for the next release
   1. If the build fails, check the error log to narrow down the reason
   2. Sometimes one of the multiple build tests will randomly fail due to issues in Travis CI so if you see just one build failure and no clear error message it may be a random issue. Add a comment so the reviewer for your change can re-run the build for you, or close the PR and re-submit and that typically clears the issue
-
-4. If you've included your mailing address in Step 1, you'll be receiving a [Limited Edition Mattermost Mug](http://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request has been accepted
+  3. A member of the core team may contact you via the [Mattermost coreteam channel][core-channel-invite] to discuss your PR. Please check this channel
+  frequently while you have PR's open.
```

Comment:
Most likely core team would reply back on the PR rather than in core team site, since we'd want the feedback as part of the PR thread, so not sure this step is needed.